example/main.tf

```diff
@@ -2,14 +2,19 @@ module "cloudfront_auth" {
   source = "../"
 
   auth_vendor             = "cognito"
+  authz                   = "2"  # Is this a number or a name?
   cloudfront_distribution = "private.example.com"
+  cloudfront_aliases      = ["secret.example.com"]
+  base_uri                = "https://auth-xxxxxx.auth.us-east-1.amazoncognito.com"
```

Comment:
This is the base uri for congnito [idp](https://github.com/mslipets/terraform-cloudfront-auth/blob/8b28e4b72f88ce62f87657e8f74067ff48a828c1/variables.tf#L29) settings endpoint
see https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-app-idp-settings.html